docs/application/dotnet/guides/security/requesting-permissions.md

```diff
@@ -141,6 +141,94 @@ To check whether an application has permission to use a privilege, and to reques
 
     If the decision is definitive, any subsequent `RequestPermission()` calls result in an immediate response with an appropriate result: `AllowForever` or `DenyForever`. However, the user can change the status of privacy-related privileges later by modifying the privacy settings on the device. For this reason, the application must always check the status of privacy-related privileges before using protected functionality.
 
+## Requesting Multiple Permissions
+
+This section describes how to check and request multiple privileges in a single API call.
+
+> **Note**
+>
+> Multiple privileges in single API call are supported from Tizen 5.5.
+
+To check whether an application has permission to use a privilege, and to request permission if required:
+
+1. Check whether an application has permission to use privileges, using the `CheckPermissions()` metchod:
```

Comment:
..." to use a particular privilege, use"
suggests that `CheckPermissions()` supports only one privilege check but it can check permission for multiple privileges at once.
I will update it to:
"To verify whether an application has permission to use privileges, use `CheckPermissions()`"